integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -187,7 +197,7 @@ def _compile_from_path(sm_path):
     # Create a saved model for these target backends to avoid a race condition
     # when running a test suite.
     # TODO(meadowlark): Remove this once we have a TfLiteCompiledModule.
-    sm_path = os.path.join(artifacts_dir, f"saved_model__{backends_string}")
+    sm_path = os.path.join(artifacts_dir, _specify_backends("saved_model"))
```

Comment:
inconsistent ' vs " compared to above.

Reply:
The file uses `"` by default, but I use `'` to let me specify a string within a `"`-style `f-string` on line 181.